src/app/components/accordion-group/accordion-group.tsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import $ from '~/helpers/$';
+import {
+    Accordion,
+    AccordionItem,
+    AccordionItemHeading,
+    AccordionItemButton,
+    AccordionItemPanel
+} from 'react-accessible-accordion';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {faChevronUp} from '@fortawesome/free-solid-svg-icons/faChevronUp';
+import {faChevronDown} from '@fortawesome/free-solid-svg-icons/faChevronDown';
+import './accordion-group.scss';
+
+type FunctionOrFalse = false | ((n: unknown) => void);
+
+function useChevronDirection(
+    forwardOnChange: FunctionOrFalse,
+    preExpanded: unknown[]
```

Comment:
Is this and the onChange param ever not an array of strings? I'm wondering if we can solidify this type.